src/main/java/keycontacts/logic/commands/CancelLessonCommand.java

```diff
@@ -0,0 +1,46 @@
+package keycontacts.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static keycontacts.logic.parser.CliSyntax.PREFIX_DATE;
+import static keycontacts.logic.parser.CliSyntax.PREFIX_START_TIME;
+
+import java.time.LocalDate;
+
+import keycontacts.logic.commands.exceptions.CommandException;
+import keycontacts.model.Model;
+import keycontacts.model.lesson.Time;
+
+/**
+ * Cancels a currently schedule lesson.
+ */
+public class CancelLessonCommand extends Command {
+
+    public static final String COMMAND_WORD = "cancel";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Cancels the lesson identified by the date and startTime.\n"
```

Comment:
I think it may be better to put startTime as start time instead since this is user facing?